MITADS/opensubtitles_exporter.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env python3
+
+from typing import Pattern
+import re
+from pathlib import Path
+from unidecode import unidecode  
+import time
+
+
+mapping_normalization = [
+
+  ##L'Ordine delle entry (replace e sub con regex) è significativo!!!
+
+  ##se la frase contiene parole con underscore o  @, allora non fa parte di una conversazione
+  ##[ re.compile('(^|(.*\s))\w+(_|@|®|\+|\{|\})\w+((\s.*)|$)'), u'' ],
+
+  ##se la frase inizia con un numero , viene eliminata
+  [ re.compile('^\d+(.*)'), u'' ],  
+
+  ##tutto ciò è contenuto tra paretesi va eliminato, non fa parte di una conversazione    
+  [ re.compile('(\(|\[|{)[^(\)|\]|})]*(\)|\]|})'), u'' ],
+
+  ## potrebbero essere parentesi annidate(ci sono nel dataset), lo lancio due volte
+  [ re.compile('(\(|\[|{)[^(\)|\]|})]*(\)|\]|})'), u'' ],
+
+  #### testo compreso tra separatori 
+  #[ re.compile('(\-|=)[^(\-|=)]*(\-|=)'), u'' ],
+  
+  ##replace caratteri, sono presenti nelle frasi
+  [ u'-' , u' ' ],
+  [ u'=' , u' ' ],
+  [ u'_' , u' ' ],
+  [ u'–' , u' ' ],##trattino differente  
+  [ u'+' , u'' ],
+  [ u'(' , u'' ],
+  [ u'|' , u'' ],
+  [ u'—' , u' ' ],  
+  [ u')' , u'' ],
+  [ u'[' , u'' ],
+  [ u']' , u'' ],
+  [ u'~' , u'' ],
+  [ u'=' , u'' ],  
+  [ u'*' , u'' ],  
+  [ u'/' , u' ' ],  
+  [ u'¢' , u'c' ],    
+  ##apici doppi , rimuovo sempre    
+  [ u'"' , u'' ],
+  [ u'¨' , u'' ],
+  [ u'^' , u'' ],
+  
+  
+  ##sequenza di punti
+  [ re.compile('\.+'), u'.' ],
+  ##sigle e testi con punti
+  [ re.compile('\n|\t|\r'), u' ' ],
+  [ re.compile('\s+'), u' ' ],
+
+  ##accentate maiuscole
+  [ u'È' , u'è' ],
+  [ u'E\'\s' , u'è' ],  
+
+  ##apostrofi, sono presenti 
+  [ u'´' , u'\'' ],
+  [ u'`' , u'\'' ],
+  [ u'\'\'' , u'\'' ],  
+
+  ##testi racchiusi tra apici singoli - in conflitto con le accentate, rimuovo questo tipo di apici
+  [ re.compile('(\s|^)(\')([^\']*)(\')(\s|$)'), r'\3' ],
+
+  ## punteggiatura non significativa alla fine del testo  
```

Comment:
Can we have an example sentence for this?

Reply:
I removed it after some test